### PR TITLE
Published Postman collection for PosDealer API

### PR DESCRIPTION
**Pull request by technical-writer-fabry**
- [ ] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

**Review by FlorianStadlberger — APPROVED** [review]

File: portal/2023-08-04-changelog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Start by retrieving relevant data for your PosOperators. You may invite new PosOperators and manage their outlets.
+Utilize the check of available entitlements and use rollout plans to transfer entitlements.
+
+For those who prefer a graphical user interface, all operations are accessible in the fiskaltrust.Portal.
+
+## where to begin?
+
+Discover the convenience and usability of our [PosDealer API Postman Collection](https://posdealer-samples.docs.fiskaltrust.cloud/ "PosDealer API") now!  
+Your feedback is crucial in helping us improve. Reach out to us at [feedback+portal](feedback+portal@fiskaltrust.cloud).
```
Comment (FlorianStadlberger) [inline portal/2023-08-04-changelog.md:25]: Correct email syntax:
[Contact Us](mailto:admin@w3schools.io)

Comment (FlorianStadlberger) [reply portal/2023-08-04-changelog.md:25]: Damn it. Markdown needs to be escaped in the comments. Here the link instead https://www.w3schools.io/file/markdown-links/

File: portal/2023-08-04-changelog.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+authors: posdealer
+slug: portal/posdealerapi
```
Comment (TSchmiedlechner) [inline portal/2023-08-04-changelog.md:3]: Slugs need to be unique, please select something more clear here.
E.g.:
```suggestion
slug: portal/posdealerapi-postman-release
```

File: portal/2023-08-04-changelog.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+tags: [Portal, PosDealerAPI]
```
Comment (TSchmiedlechner) [inline portal/2023-08-04-changelog.md:4]:
```suggestion
tags: [Portal, PosDealer API, Samples]
```

**Comment by TSchmiedlechner** [comment]
@technical-writer-fabry please use meaningful names for the PR titles and branch names, e.g. in this case:
**Branch name**: `user/rfa/posdealerapi-postman`
**PR title**: `Published Postman collection for PosDealer API`

File: portal/2023-08-04-changelog.md (hunk 1 above)
Comment (technical-writer-fabry) [reply portal/2023-08-04-changelog.md]: changed the link

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**